src/pages/about/index.mdx

```diff
@@ -24,98 +24,73 @@ import { Callout } from 'nextra/components'
 
 # About
 
-☕ Coffee? Yes, but we brew AI too.
+🖖 High-level summary: We're brewing AI in public by solving really real AI problems.
 
-![Eniac](./_assets/eniac_computer.jpg)
-_[Eniac](https://www.computerhistory.org/revolution/birth-of-the-computer/4/78), the World's First Computer (Photo courtesy of US Army)_
-
-Homebrew is an AI research lab. Our mission is to build human-augmenting AIs that run on energy-efficient hardware.
-
-<Callout>
+<Callout type="info" emoji="ℹ️">
 We were inspired by the [Homebrew Computer Club](https://en.wikipedia.org/wiki/Homebrew_Computer_Club), an early computer hobbyist group from 1975 to 1986 that led to Apple and the personal computer revolution.
 </Callout>
 
-## What drives us
-
-![Tesla](./_assets/tesla_in_his_lab.webp)
-_Nikola Tesla sitting in his Colorado Springs laboratory._
+![Homebrew_Computer_Club_Invitation](./_assets/Invitation_to_First_Homebrew_Computer_Club_meeting.jpg)
+_This is the first invitation from Steve Jobs to the Homebrew Computer Club, echoing the spirit of Homebrew Research: Are you experimenting with AI?_
 
-Like [electricity](https://www.economist.com/leaders/2000/08/03/the-electric-revolution) powering homes or space travel making the impossible possible, AI is our next leap forward. We see AI as a tool to make us smarter, kinder, and more capable.
+## About Us
 
-Like fire that cooked our food and forged our tools, or electricity that powered our cities and connected our world, AI sparks both wonder and worry. We believe that [AI augmenting human creativity](https://link.springer.com/article/10.1007/s11023-024-09677-x) will help us change the future we've yet to imagine.
+<Callout emoji="👾">
+Homebrew is a local AI company.
+</Callout>
 
-### Human Augmentation
+What that means, in case you've been living under a rock (which, given the state of things, might not be such a bad idea), is that we're a bunch of well-meaning fools trying to create AI that stays put. Like a good dog. 
```

Comment:
Maybe it's a cultural thing, but I read this as a bit condescending.

Reply:
I reread the whole page and, I agree that it's not the best way to tell the story with our tone.